source/_integrations/easyenergy.markdown

```diff
@@ -47,6 +47,18 @@ prices for electricity that you use (buy) or return (sell). Every day around
 - Time of day when the price is highest
 - Time of day when the price is at its lowest
 - Percentage of the current price compared to the maximum price
+- Number of hours with the current price higher or lower
+
+Entities with the number of hours, indicates how many hours there are with a price
+**above** or **below** the current hourly price. If we take the graph below as an example
+and it is 00:30, then there are 8 hours below the current price and 4 hours above the
+current price. With this information you could switch a device at the X cheapest number
+of hours during the day.
+
+<p class='img'>
+  <img src='/images/integrations/easyenergy/pricegraph.png' alt='Screenshot showing energy price graph.'>
```

Comment:
The graph shown isn't possible to make with the built-in provided graphing. Adding graphs like these, make users think, however, it is possible. I would not recommend it for that reason.

Reply:
It's a graph from easyEnergy that I made a screenshot of. I also added it not with the intention that users can create this, but as a visual image to better understand how the new entities work 🤷🏻‍♂️

Reply:
That I didn't realize, yeah clarifying might help there 👍